### ISSUE TITLE
btrack on GPU

### ISSUE THREAD

**Issue by Ciro1990** [quantumjot/BayesianTracker#124]
Dear Alan,

first of all, thanks for creating and sharing this fantastic tool. I was wondering if btrack on GPU is now available and is working. 

Thanks a lot!
Ciro

**Comment by quantumjot** [comment]
Hi Ciro,
Haven't had much time to work on this unfortunately.  Is there a reason why you need the GPU accelerated version?

Related #39

**Comment by quantumjot** [comment]
Yes - arriving at a good configuration can be challenging. Are you able to share any extra details, like a log or a short movie (e.g. #125)? - I might be able to give you some quick advice.
